e2e/test/scenarios/dashboard-filters/parameters.cy.spec.js

```diff
@@ -803,24 +754,23 @@ describe("scenarios > dashboard > parameters", () => {
           .should("be.visible");
       });
 
-      it("should undo parameter wiring when 'Undo auto-connection' is clicked", () => {
-        const cards = [
-          {
-            card_id: ORDERS_BY_YEAR_QUESTION_ID,
-            row: 0,
-            col: 0,
-            size_x: 5,
-            size_y: 4,
-          },
-          {
-            card_id: ORDERS_COUNT_QUESTION_ID,
-            row: 0,
-            col: 4,
-            size_x: 5,
-            size_y: 4,
-          },
-        ];
+      it("should add and wire a card without a failing request during editing (metabase#36469)", () => {
+        createDashboardWithCards({ cards }).then(dashboardId =>
+          visitDashboard(dashboardId),
+        );
+
+        editDashboard();
+        setFilter("Time", "Relative Date");
+        selectDashboardFilter(getDashboardCard(0), "Created At");
+        saveDashboard();
+
+        editDashboard();
 
+        addCardToDashboard("Orders, Count");
+        getDashboardCard(2).findByText("Count").should("exist");
```

Comment:
We expect the column header to exist, whereas when there's an error, it doesn't.